src/crawlee/http_crawler/__init__.py

```diff
@@ -1,4 +1,10 @@
+from crawlee.http_clients import HttpCrawlingResult
+from crawlee.static_content_crawler._static_crawling_context import HttpCrawlingContext
+
 from ._http_crawler import HttpCrawler
-from ._http_crawling_context import HttpCrawlingContext, HttpCrawlingResult
 
-__all__ = ['HttpCrawler', 'HttpCrawlingContext', 'HttpCrawlingResult']
+__all__ = [
+    'HttpCrawler',
+    'HttpCrawlingContext',
+    'HttpCrawlingResult',
```

Comment:
Is it necessary to expose the `HttpCrawlingResult` to the public?

Reply:
It was like this before the change, so I did not want to make any unnecessary breaking changes.

Reply:
It is exposed for people who want to implement their own `BaseHttpClient` subclass.